linux_files/profile

```diff
@@ -40,4 +40,11 @@ fi
 export DISPLAY=:0
 
 # enable external libgl 
-export LIBGL_ALWAYS_INDIRECT=1
\ No newline at end of file
+export LIBGL_ALWAYS_INDIRECT=1
+
+# does .firstrun exist? if not, run setup and touch .firstrun
+ 
+if [ ! -f ~/.firstrun ]; then
+    /bin/bash /etc/setup
+    touch ~/.firstrun
+fi
```

Comment:
Add a script for update checker later... like oh my zsh style, it's okay for now